web/api/formatters/json_wrapper.c

```diff
@@ -32,14 +32,20 @@ void rrdr_json_wrapper_begin(RRDR *r, BUFFER *wb, uint32_t format, RRDR_OPTIONS
 
     RRDHOST *host = r->st->rrdhost;
     buffer_sprintf(wb, "   %slabels%s: {\n", kq, kq);
+
     int count = 0;
-    netdata_rwlock_rdlock(&host->labels_rwlock);;
+    netdata_rwlock_rdlock(&host->labels_rwlock);
     for (struct label *label = host->labels; label; label = label->next) {
         if(count > 0) buffer_strcat(wb, ",\n");
-        buffer_sprintf(wb, "      %s%s%s: %s%s%s", kq, label->key, kq, sq, label->value, sq);
+
+        char value[CONFIG_MAX_VALUE + 1];
```

Comment:
If this is the target of the expansion it does not include space for the extra escaping characters.

Reply:
It stops copying data when maximum length is reached.

Reply:
So it will output a truncated label - which is not correct either.